doc/specs/OOBE-Dialog.md

```diff
@@ -0,0 +1,112 @@
+|8 | Automatically save changes to settings as they are made in the OOBE. | P1 |
+|9 | In the PowerToys settings “General” menu, add a button to re-launch the OOBE dialog under the “Check for updates” button. | P0 |
+|10 | The window is resizable and the content is responsive to being resized. | P0 |
+|11 | The OOBE dialog should have a preset minimum window size to ensure all content can be viable with the need to scroll the page. | P1 |
+
+#### 3.1.2. Page Layout
+
+This section describes the content that should be displayed on a given page for each PowerToys utility.
+
+| No. | Requirement | Pri |
+| - | - | - |
+|1 | Each page of the OOBE dialog should focus on the details and settings of a single PowerToy. | P0 |
+|2 | A page should include textual information that gives an overview of its PowerToy's functionality. | P0 |
+|3 | A page should have a dedicated "How to run/launch" section for explaining the shortcut or actions needed to utilize the PowerToy, emphasizing that users are encouraged to "try it out" while in the OOBE. | P0 |
+|4 | A page should include a hero image/GIF that provides a visual representation of its PowerToy's core functionality. | P0 |
+|5 | A page should include a link to that PowerToys overview page on Microsoft Docs. | P1 |
+|6 | Include a &quot;Settings&quot; button that deep links to the PowerToy's settings page. | P1 |
+|7 | Include a &quot;Pro Tips&quot; section on the page that highlights a useful, yet often undiscovered feature for its PowerToy. | P3 |
+
+#### 3.1.3. Welcome/Overview Page Content
+
+This section describes the content that should be displayed on the Welcome/Overview Page of the OOBE window. [Figure 5.1.2](#512-oobe-welcomeoverview-page) provides a visual of the layout.
+
+| No. | Requirement | Pri |
+| - | - | - |
+|1 | A high level, textual overview of the PowerToys application. | P0 |
+|2 | Link to PowerToys General Settings. | P0 |
+|3 | Link to PowerToys overview on Microsoft Docs. | P0 |
+|4 | Link to PowerToys releases page to showcase latest update changes. | P1 |
+
+## 4. Measure Requirements
+
+| No. | Requirement | Implication | Pri |
+| --- | --- | --- | --- |
+|1 | Date/Time of first-run | Helps to categorize usage and retention trends across various groups of new users. | P0 |
+|2 | OOBE sections viewed | Used to gauge how far users progress through the OOBE dialog. Helps guide decisions on further enhancements to improve retention. | P0 |
+|3 | Access to linked settings pages | Used to gauge whether users would find value in having quick settings access for particular utilities. | P1 |
+|4 | PowerToys launched while OOBE window is active. | Used to track user engagement with the various PowerToys while exploring the content in the OOBE. | P1 |
+|5 | Screen size | Gives crucial information for considerations related to minimal/maximum window size needed for displaying content. | P2 |
```

Comment:
This information might be already available as part of the system telemetry.

Reply:
Sounds good, this shouldn't require any work then.